shared/src/balancer/pool_storage.rs

```diff
@@ -192,6 +219,11 @@ impl PoolStorage {
                     .insert(pool_id);
             }
         }
+        tracing::info!(
+            "inserted {} balancer pool creation events. Synced to block {}",
+            num_events,
+            self.last_event_block()
```

Comment:
Is this accurate? If we sync to block X but the last pool was created at some other block Y won't this log be misleading?

Reply:
Yes... although it does say up till which block we have pools. If this is too misleading or confusing, I would rather just remove it.

Reply:
It says `Synced to block` which is not true, since the latest pool we have is not necessarily on the latest block we've synced.

Reply:
Yea I see the problem. This log has been removed.